deploy/kubernetes/examples/demo-snapshotclass-using-secret-from-configuration.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: snapshot.storage.k8s.io/v1beta1
```

Comment:
would `/v1` work?

Reply:
IDK

Reply:
could you check? we don't want to add the old version if we don't have to

Reply:
a ticket number is also fine

Reply:
blocked on CSI-2873.
v1 is for GA  external-snapshotter.

Reply:
if we want to rely on CSI-2873 for bumping to v1, it should be in the ticket **description**

Reply:
done